packages/@react-spectrum/button/chromatic/Button.chromatic.tsx

```diff
@@ -58,17 +58,17 @@ storiesOf('Button', module)
       <Flex gap="size-200">
         <Button
           variant="primary">
-          {0} Dogs
+          <Text>{0} Dogs</Text>
```

Comment:
Note: people who did something similar in their Button components will experience the unset `min-width` applied by the new `iconOnly` class. I figure this is ok since correct usage of Button is to wrap the contents in a `Text` element if it isn't a plain string

Reply:
I added an extra conditional for iconOnly styling in #3503 so that it doesn't break these implementations